website/src/components/Features.tsx

```diff
@@ -31,6 +57,38 @@ export function Features() {
                 </li>
               ))}
             </ul>
+            {/* email form */}
+            <form className="mt-10 px-20" onSubmit={handleEmailSubmission}>
+              <h3 className="text-lg font-medium tracking-tight text-black">
+                {t('feature.try')}
+              </h3>
+              <div className="mt-4 sm:relative sm:flex sm:items-center sm:py-0.5 sm:pr-2.5">
```

Comment:
**suggestion:** using relative positioning with a relative background is probably not the best way to design this button. 

You should look at flex positioning with the parent element having the border